app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -123,6 +123,7 @@ a {
   width: 44px;
   height: 34px;
   margin: 0 5px 0 0 !important;
+  max-width: fit-content;
```

Comment:
`fit-content` does not work for firefox, as it is considered experimental - https://developer.mozilla.org/en-US/docs/Web/CSS/max-width.
Increasing the right margin one row above seems to also fix it.

Reply:
@tbrisker I don't think `margin` is the problem here.
I picked `max-width` because PF4 seem to set all to those to `max-width: 100%;`:
```css
img,
embed,
iframe,
object,
audio,
video {
  max-width: 100%;
  height: auto; }
```

Reply:
changing it into `max-width: inherit;` as it also works :+1: